hedyorg/hedy
# 🪲 Fix content html elements parsing

This PR ensures that the if the content of Hedy contains html parts, they will be parsed correctly and not displayed as plain text.

Fixes #5901 

**How to test**
Log in as a teacher and open the for-teachers tab. Ensure that the all the links on this page are displayed as links and not as <a> tags.
